google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITObjectTest.java

```diff
@@ -1069,28 +1008,6 @@ public void testCopyBlobFail() {
     }
   }
 
-  @Test
-  public void testReadAndWriteChannels() throws IOException {
-    String blobName = "test-read-and-write-channels-blob";
-    BlobInfo blob = BlobInfo.newBuilder(BUCKET, blobName).build();
-    byte[] stringBytes;
-    try (WriteChannel writer = storage.writer(blob)) {
-      stringBytes = BLOB_STRING_CONTENT.getBytes(UTF_8);
-      writer.write(ByteBuffer.wrap(BLOB_BYTE_CONTENT));
-      writer.write(ByteBuffer.wrap(stringBytes));
-    }
-    ByteBuffer readBytes;
-    ByteBuffer readStringBytes;
-    try (ReadChannel reader = storage.reader(blob.getBlobId())) {
-      readBytes = ByteBuffer.allocate(BLOB_BYTE_CONTENT.length);
-      readStringBytes = ByteBuffer.allocate(stringBytes.length);
-      reader.read(readBytes);
-      reader.read(readStringBytes);
-    }
-    assertArrayEquals(BLOB_BYTE_CONTENT, readBytes.array());
-    assertEquals(BLOB_STRING_CONTENT, new String(readStringBytes.array(), UTF_8));
-  }
-
   @Test
   public void testReadAndWriteChannelWithEncryptionKey() throws IOException {
```

Comment:
Asking for opinions here, there are 3 tests that don't seem to fit nicely into other Read/Write channel tests to move.
- testReadAndWriteChannelWithEncryptionKey
- testReadAndWriteChannelsWithDifferentFileSize
- testReadAndWriteCaptureChannels

Any opinions on what to do with these tests?

Reply:
* `testReadAndWriteChannelWithEncryptionKey` I think is okay to remove, we now have other tests which actually verify args like encryption key are wired through.
* `testReadAndWriteChannelsWithDifferentFileSize` semi useful, but written in such a way that it's painful. We can refactor this one at a later point.
* `testReadAndWriteCaptureChannels` unfortunately we need to keep this, this is verifying that a "captured" channel can be restored - despite the fact that it's often confusing.